### PR TITLE
[Feature] Enhance metadata menu tree lazy load mock and suffix button style

### PR DESCRIPTION
**Pull request by zhangmo8**
### Purpose

MenuTree lazyLoad mock finish & interaction complete

suffix button style finish

截屏2023-11-01 23 02 18

### CODE REVIEW

**Comment by zhangmo8** [comment]
#85

**Review by labbomb — APPROVED** [review]
LGTM.

File: paimon-web-ui-new/src/api/models/catalog/index.ts [hunk 1]
```diff
@@ -29,4 +29,11 @@ export const getAllCatalogs = () => {
   return httpRequest.get<any, Catalog[]>('/catalog/getAllCatalogs')
 }
 
+/**
+ * # Get database by catalog id
+ */
+export const getDatabaseByCatalogId = (id: number) => {
+  return httpRequest.get<any, Catalog[]>(`/database/getDatabasesById/${id}`)
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/api/models/catalog/index.ts:36]: Why not `/database/getDatabasesByCatalogId/${id}`? Or `/database/getDatabasesByCondition/${id}`

**Review by SteNicholas — COMMENTED** [review]
@zhangmo8, thanks for contribution. I left some comments for this pull request. PTAL.

File: paimon-web-ui-new/src/api/models/catalog/index.ts (hunk 1 above)
Comment (zhangmo8) [reply paimon-web-ui-new/src/api/models/catalog/index.ts:36]: @hunter-cloud09

File: paimon-web-ui-new/src/api/models/catalog/index.ts [hunk 2]
```diff
@@ -29,4 +29,11 @@ export const getAllCatalogs = () => {
   return httpRequest.get<any, Catalog[]>('/catalog/getAllCatalogs')
 }
 
+/**
+ * # Get database by catalog id
+ */
+export const getDatabaseByCatalogId = (id: number) => {
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/api/models/catalog/index.ts:35]:
```suggestion
export const getDatabasesByCatalogId = (id: number) => {
```

File: paimon-web-ui-new/src/views/metadata/components/menu-tree/index.tsx [hunk 3]
```diff
@@ -42,29 +43,79 @@ export default defineComponent({
...
+        default: () => h(NIcon, null, {
+          default: () => h(Add)
+        })
+      }) : undefined
     }
 
     onMounted(catalogStore.getAllCatalogs)
 
+    const onLoadMenu = async (node: TreeOption) => {
+      const loadFn = node.type === 'catalog' ? catalogStore.getDatabaseByCatalogId : catalogStore.getTableByDataBaseId
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/menu-tree/index.tsx:89]:
```suggestion
      const loadFn = node.type === 'catalog' ? catalogStore.getDatabaseByCatalogId : catalogStore.getTablesByDataBaseId
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
PTAL.

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request merged**